src/ray/common/task/task_spec.cc

```diff
@@ -145,8 +145,8 @@ const ResourceSet &TaskSpecification::GetRequiredPlacementResources() const {
 }
 
 bool TaskSpecification::IsDriverTask() const {
-  // Driver tasks are empty tasks that have no function ID set.
-  return FunctionDescriptor().empty();
+  return FunctionDescriptor()->Type() ==
+         FunctionDescriptorType::kDriverFunctionDescriptor;
```

Comment:
I think we can add `DRIVER_TASK` type to `TaskType` in `common.proto`. Then we don't need to define a special `DriverFunctionDescriptor` type.

Reply:
Good suggestion! I haven't noticed that there has a TaskType in TaskSpec.